science/inspectrum/Portfile

```diff
@@ -6,11 +6,11 @@ PortGroup           github 1.0
 PortGroup           qt5 1.0
 PortGroup           app 1.0
 
-github.setup        miek inspectrum d6115cb458068fa64c7bbd9020bcbab3373fee0e
-version             20191102
-checksums           rmd160 d0dbbda96fe5ca060aada402d986b34c4a63b67b \
-                    sha256 5497f4357f8603634ee0a43dc4eae9997ab35436438078c4dd6b0c5f01944836 \
-                    size   109324
+github.setup        miek inspectrum v0.2.3
+version             0.2.3
```

Comment:
This is normally done with one line as follows:

`github.setup        miek inspectrum 0.2.3 v
`